.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -3,15 +3,31 @@ description: Tell us about something that's not working the way we (probably) in
 labels: ["Platform: Cocoa", "Type: Bug"]
 body:
   - type: dropdown
-    id: environment
+    id: platform
     attributes:
       label: Platform
       description: Which platform do you use?
+      multiple: true
       options:
         - iOS
```

Comment:
`m`: I think we should also add iPadOS to be sure - now that multi-select is possible it makes even more sense IMO